### DISCUSSION TITLE
[问题] --watch 模式下每修改一次代码自动编译慢

### DISCUSSION THREAD

**Discussion by q759410559** [remaxjs/remax#1584] [General]
**问题描述**
--watch 模式下每修改一次代码实时构建每次都要1分多钟，有没有什么办法优化一下，提高编译速度。 小程序80多个页面。

**示例代码**

**其他信息**
![image](https://user-images.githubusercontent.com/11240348/119316310-67afc700-bca9-11eb-9a22-0488422e3701.png)

**Comment by yesmeck** [comment]
80个页面有点多了，建议开发时根据环境变量在 app.config.js 中按需加载页面。
